.travis.yml

```diff
@@ -9,6 +9,7 @@ notifications:
 before_install:
 - go get github.com/mitchellh/gox
 script:
+- hack/ensure-kubectl-installed.sh
```

Comment:
Can you move this down right before `run-integration-tests.sh`? That would speed up failing runs a little.